### PR TITLE
Jumpbox provisioner changes

### PR DESCRIPTION
**Pull request by JermilaPaulDhas**

### CODE REVIEW

File: content-jumpbox-provisioner/cloudformation/stack.yml [hunk 1]
```diff
@@ -101,22 +117,34 @@
...
+            chmod 0644 /etc/ssh/ssh_config
+            echo "${SSHAgent}" >> /etc/profile.d/z0_ssh_agent.sh
+            chmod 0644 /etc/profile.d/z0_ssh_agent.sh
+            echo "${K8sCluster}" >> /etc/profile.d/z2_k8s_clusters.sh
+            chmod 0644 /etc/profile.d/z2_k8s_clusters.sh
+            echo 0644 "${ListClusters}" >> /usr/local/bin/list-clusters
+            chmod +x /usr/local/bin/list-clusters
+            cd /home/ec2-user
+            # Install kubectl
+            wget https://storage.googleapis.com/kubernetes-release/release/v1.7.6/bin/linux/amd64/kubectl
```
Comment (sbuliarca) [inline content-jumpbox-provisioner/cloudformation/stack.yml]: I think we should be using a newer version of kubectl, like 1.10

Comment (JermilaPaulDhas) [reply content-jumpbox-provisioner/cloudformation/stack.yml]: Done. Also updated the k8s-cli-utils https://github.com/Financial-Times/k8s-cli-utils/pull/9

File: content-jumpbox-provisioner/k8s_ssh/ssh_config [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# to the original X11 display. As virtually no X11 client supports the untrusted
+# mode correctly we set this to yes.
+        ForwardX11Trusted yes
+# Send locale-related environment variables
+        SendEnv LANG LC_CTYPE LC_NUMERIC LC_TIME LC_COLLATE LC_MONETARY LC_MESSAGES
+        SendEnv LC_PAPER LC_NAME LC_ADDRESS LC_TELEPHONE LC_MEASUREMENT
+        SendEnv LC_IDENTIFICATION LC_ALL LANGUAGE
+        SendEnv XMODIFIERS
+        
+Host *-tunnel-up.ft.com
```
Comment (sbuliarca) [inline content-jumpbox-provisioner/k8s_ssh/ssh_config]: I don't think we use these host names any more.

File: content-jumpbox-provisioner/k8s_ssh/k8s_clusters.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+echo -e \"
+~~~
+
+\e[31mKubernetes Instructions\e[39m
+
+To show available clusters:
+list-clusters
+
+To connect to a cluster:
+source cluster-login.sh upp-delivery-prod-eu
```
Comment (sbuliarca) [inline content-jumpbox-provisioner/k8s_ssh/k8s_clusters.sh]: Probably we also need to instruct them to open the outputted URL in a browser and to copy the token afterwards.

**Comment by JermilaPaulDhas** [comment]
@sbuliarca Updated the PR with the following changes:
* Removed the ssh config coz ops are not expected to login to our tunnel boxes to ssh to the worker instances
* Added instructions for Ops to copy past url and token

**Pull request merged**